build/config.gni

```diff
@@ -68,7 +68,17 @@ if (is_win) {
       assert(brave_channel == "", "Unknown channel name")
     }
   } else {
-    _channel = "Development"
+    if (brave_channel == "nightly") {
+      # Temporary support PR's Static builds on nightly channel.
+      # CI expect the installer files (*Setup.exe) in the output dir are
+      # correspond passed brave_channel. Therefore to make Static+nightly work
+      # we should set _channel to "Nightly".
+      # TODO(atuchin): revert after PRs start to use the development channel.
+      _channel = "Nightly"
```

Comment:
@bridiver
We need this until we switch to `development`. We can't "switch" channels here because
the CI now expects in `out/` files like `BraveBrowserStandaloneNightly.exe` (not `BraveBrowserStandaloneDevelopment.exe`)

IMHO we can after add an assert here like `assert(brave_channel == "", "You can't set a channel for non-release builds")`